drned-skeleton/save-default-config.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import print_function
+
+from contextlib import closing
+
+from devcli import Devcli, XDevice
+
+
+def _save_default_config(nso_device, target_device):
+    target_device.save_config()
+    #do we want nso_device.sync_to/from() here?
+
+def save_default_config(nsargs):
+    def init_cli_dev():
+        return closing(Devcli(nsargs))
+
+    def init_nso_dev(devcli):
+        return closing(XDevice(devcli.devname))
+
+    with init_cli_dev() as target_device, \
+            init_nso_dev(target_device) as nso_device:
+        _save_default_config(nso_device, target_device)
```

Comment:
On one hand, you are doing the same stuff that is done in load-default-config as well as in cli2netconf (even it is expressed somewhat differently there), so maybe the time for abstracting this; on the other hand, you are not using the `nso_device`, so maybe not yet?

Reply:
yes, i've been thinking about the same, but discarded it at least for now due to no NSO side updates...